### DISCUSSION TITLE
Support for TX as RX

### DISCUSSION THREAD

**Discussion by devanesanandrews** [ExpressLRS/ExpressLRS#1332] [Ideas]
Now a days there are a lots of 3d printed bays can be 3d printed.

In big models it would be easy to put a bay and slide the TX to be used as RX

can this feature be added?

*Instead of commenting "I want this too", consider leaving a :+1: instead, to reduce spam and clutter*

**Comment by brandonrc** [comment]
I like this idea but I think it would be a case by case thing unless you have a TX in mind that would be good as a RX? 

I think every TX will have their unique pins so to make this a general option would be asking a bit much.

**Reply by devanesanandrews** [reply]
To be more specific its for the diy TX which has more power (100mw to 2000mw)

in Future when a full telemetry is implemented it would be of great use.
